src/locale/locales/pt-BR/messages.po

```diff
@@ -7855,7 +7855,7 @@ msgstr "Verificar e-mail"
 
 #: src/components/intents/VerifyEmailIntentDialog.tsx:61
 msgid "Verify email dialog"
-msgstr ""
+msgstr "Caixa de diálogo verificar e-mail"
```

Comment:
Native speaker here.

I'm not sure what's the context for "Verify email dialog", but the translation is lexically incorrect. 

Is this something along the line of "Verify an exchange of e-mails"? If so, It's something along the line of "Verificar troca de e-mails".

Reply:
This string is the accessibility label to describe the dialog that pops up after you've clicked a link to verify your email address
https://github.com/bluesky-social/social-app/blob/b37b64fb49bd77597ac9b4aa239edc3f05bfd0e0/src/components/intents/VerifyEmailIntentDialog.tsx#L61

Reply:
@pedrochamberlain Based on the context, maybe we can use "Caixa de diálogo de verificação de e-mail" or "Caixa de diálogo da verificação de e-mail". What do you think?

Reply:
@andregamma Sorry I should have marked this resolved, I went with the suggestion [here](https://github.com/bluesky-social/social-app/pull/5256#discussion_r1752445154) of `Caixa de diálogo de verificação de e-mail`. Thanks :)

Reply:
@fabiohcnobre It should be `da verificação` rather than `de verificação`?

Reply:
@surfdude29 
I checked the rule. The correct way in this case would be 'da' because it is a specific verification that would be email. If there was no email then you could use 'de'.

Reply:
@fabiohcnobre Ah I see, thanks for the explanation :)